demos/demo.js

```diff
@@ -104,6 +105,7 @@ function Register(name, type, id) {
             case 'i8':
             case 'i16':
             case 'i32':
+            case 'i64':
```

Comment:
Note that `this._update_int` refreshes only (up to) 32-bit integers via:
```
        var value = e.reg_read_i32(this.id);
```
This will fail at updating the upper 32 bits of 64-bit registers.